less/mixins.less

```diff
@@ -1 +1,29 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+.center () {
+    display: flex;
+    justify-content: center;
+}
+
+.custom-btn(@width, @height, @bg-color, @font-color, @font-size) {
+    width: @width;
+    height: @height;
+    background-color: @bg-color;
+    color: @font-color;
+    cursor: pointer;
+    font-size: @font-size;
+    .center();
+    align-items: center;
+    border-radius: 8px;
+
+    &:hover {
+        background: @font-color;
+        color: @bg-color;
+    }
+
+}
+
+
+
```

Comment:
Good job on the custom button. You can add in a transition: all.3s ease; on the button to give a bit of timing to the color change